packages/components/src/modifiers/hds-clipboard.ts

```diff
@@ -129,15 +136,35 @@ export const copyToClipboard = async (text, target) => {
 // because it's quite simple in its logic, and doesn't require injecting services
 // see: https://github.com/ember-modifier/ember-modifier#function-based-modifiers
 
-export default modifier((element, positional, named) => {
+export default modifier((element, _positional, named) => {
   assert(
     '`hds-clipboard` modifier - the modifier must be applied to an element',
     element
   );
 
-  const { text, target, onSuccess, onError } = named;
+  const {
```

Comment:
For readability, better to move these types to a standalone type declaration (in this way, it can also be imported by the consumers of the modifier)